test/integration/targets/dnf/tasks/dnf.yml

```diff
@@ -241,6 +274,8 @@
     state: present
   register: dnf_result
 
+- debug: var=dnf_result
```

Comment:
I wonder if we should put something like `verbosity: 1` for debug prints?

Reply:
@mkrizek that was actually an accidental left-over for when I was working on this, it's not needed for the tests